charts/cray-import-config/values.yaml

```diff
@@ -20,8 +20,12 @@
 # OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
+#
 
-# The image that contains the config content (based on cf-gitea-import)
+# Full image including repo, e.g. dtr.dev.cray.com/cray/config-image:1.2.3
```

Comment:
Removing referenced to dev.cray.com addresses was deliberate as that domain was retired. I am guessing there should not have been any changes in this file.